support-workers/src/main/scala/com/gu/zuora/productHandlers/ZuoraPaperHandler.scala

```diff
@@ -38,6 +38,7 @@ class ZuoraPaperHandler(
       account.value,
       sub.value,
       state.firstDeliveryDate,
+      state.similarProductsConsent,
```

Comment:
Would this need to change in future PR for the Observer?

Reply:
We will still have to pass it through for Observer but it will be a scala `None`